src/Buildalyzer/ProjectAnalyzer.cs

```diff
@@ -170,8 +172,22 @@ public IAnalyzerResults Build(string[] targetFrameworks, BuildEnvironment buildE
                 GetEffectiveEnvironmentVariables(buildEnvironment),
                 Manager.LoggerFactory))
             {
+                processRunner.Exited += () =>
+                {

```

Comment:
I'm not sure if there is a convention specified for this, but I prefer a method being assigned here.

Reply:
I've changed these event handlers to local functions 👍